config/verificationMail.js

```diff
@@ -63,7 +63,7 @@ const config = {
     const template = Handlebars.compile(source);
     const { username, email } = params;
     const tokenToSend = await jwt.sign({ email }, process.env.SECRET);
-    const url = `${process.env.SERVER_ADDRESS}/api/v1/update_password/${tokenToSend}`;
+    const url = `${process.env.FRONT_END_URL}/update_password?token=${tokenToSend}`;
```

Comment:
@shemaeric  I think the whole URL should be made an environment variable🤔 And the endpoint should use a dash instead of an underscore to be more friendly

Reply:
Thank you @avpaul  but I don't think there is the main reason for adding the whole URL to the environment, as the URL which is in the environment is the root URL for our front-end, 

If we do so, it means this environment variable will not be reusable.

Reply:
👌